src/conventions/h-group/hanabi-logic.js

```diff
@@ -60,7 +60,7 @@ export function stall_severity(state, player, giver) {
 	if (player.thinksLocked(state, giver))
 		return 3;
 
-	if (state.screamed_at)
+	if (state.screamed_at || (state.dda !== undefined && !player.thinksLoaded(state, giver, { assume: false }) && player.thoughts[player.chop(state.hands[giver]).order].inferred.has(state.dda)))
```

Comment:
The giver won't have a chop if they're locked, so we probably need an extra check here. Also, shouldn't it be if the `possible` identities contain the discarded card and not the `inferred`?

Reply:
Added a check for having a chop and used the `possible` identities. I suspect for unclued cards which aren't finessed these are usually the same.